src/stream/src/executor/asof_join.rs

```diff
@@ -183,18 +148,14 @@ impl<K: HashKey, S: StateStore, const T: AsOfJoinTypePrimitive> AsOfJoinExecutor
         input_r: Executor,
         params_l: JoinParams,
         params_r: JoinParams,
-        null_safe: Vec<bool>,
```

Comment:
I forgot the purpose of `null_safe`. Why is it removed, should we also remove it for hash join?